fastlane_bot/events/async_event_update_utils.py

```diff
@@ -79,57 +148,42 @@ async def get_token_and_fee(
                 tkn1 = connector_token
             elif tkn1 == "0x1F573D6Fb3F13d689FF844B4cE37794d79a7FF1C":
                 tkn0 = connector_token
-            # tkn0 = (
-            #     connector_token
-            #     if tkn0 != "0x1F573D6Fb3F13d689FF844B4cE37794d79a7FF1C"
-            #     and connector_token != "0x1F573D6Fb3F13d689FF844B4cE37794d79a7FF1C"
-            #     else tkn0
-            # )
-            # tkn1 = (
-            #     connector_token
-            #     if tkn1 != "0x1F573D6Fb3F13d689FF844B4cE37794d79a7FF1C"
-            #     and connector_token != "0x1F573D6Fb3F13d689FF844B4cE37794d79a7FF1C"
-            #     else tkn1
-            # )
-            # if address in [
-            #     "0x8df51A9714aE6357a5B829CC8d677b43D7e8BD53",
-            #     "0x8df51A9714aE6357a5B829CC8d677b43D7e8BD53",
-            #     "0x079cA3f710599739a22673c2856202F90D3A8806",
-            # ]:
-            #     print(
-            #         f"\n#2 connector_token: {connector_token}, anchor: {anchor}, tkn0: {tkn0}, tkn1: {tkn1}"
-            #     )
-            #     raise Exception("test")
-        cid = str(event["args"]["id"]) if exchange_name == "carbon_v1" else None
-
-        return exchange_name, address, tkn0, tkn1, fee, cid, anchor
+
+        # bookmark
+        strategy_id = str(event["args"]["id"]) if ex.is_carbon_v1_fork else 0
+        pool_info = {
+            "exchange_name": exchange_name,
+            "address": address,
+            "tkn0_address": tkn0,
+            "tkn1_address": tkn1,
+            "pair_name": f"{tkn0}/{tkn1}",
+            "fee": fee,
+            "strategy_id": strategy_id
+        }
+        carbon_v1_forks = [CARBON_V1_NAME] if ex.is_carbon_v1_fork else []
```

Comment:
Shouldn't this be chain-dependent (or passed as input from the caller, ultimately depending on the chain)?
Otherwise, what exactly has been achieved here, as a single carbon deployment is eventually handled?

Reply:
oh man, good catch! I just pushed a fix

Reply:
Sure thing man, but doesn't that still yield a list of one item, i.e., a single carbon deployment?

Reply:
Yes but in this case that is the intent. If you look at the function to get the cid, all we need is for the exchange_name to exist in the list passed to that function - typically this is an instance of the config object, but we dont actually need the config object in this case because if the instance of the exchange object is either a fork of carbon, or carbon, then this satisfies the purpose of checking the list of carbon forks (ie we can be sure that it is indeed a carbon fork without needing the config object).